scripts/scil_validate_bids.py

```diff
@@ -17,9 +18,22 @@
 import coloredlogs
 
 from scilpy.io.utils import (add_overwrite_arg, add_verbose_arg,
+                             assert_inputs_exist,
                              assert_outputs_exist)
 
 
+conversion = {"i": "x",
+              "i-": "x-",
+              "j": "y",
+              "j-": "y-",
+              "k": "z",
+              "k-": "z-",
+              "LR": "x",
```

Comment:
Sorry, this passed over my head completely. Those orientations (AP, PA and the others) are affine dependent, meaning they change if the reference frame of the image is LAS, RAS or others. My problem with bvecs and the Phase Direction metadata fields comes from that (I considered AP being y-, since it is in RAS)